### ISSUE TITLE
request_scope doesn't work with auto_bind=False?

### ISSUE THREAD

**Issue by netijoe** [matyasrichter/fastapi-injector#20]
First of all - great addition to the injector library. Nice work!
I've found an issue though. When I use `auto_bind=False` option it doesn't work with `request_scope`. 
I have a basic example here:
```python
class A:
    @inject
    def __init__(self):
        self.a = random.randint(0, 1000)

class B: 
    @inject
    def __init__(self, a: A):
        self.a = a.a

def config(binder: Binder):
    binder.bind(A, to=A)
    binder.bind(B, to=B, scope=request_scope)

inj= Injector([config], auto_bind=False)

app = FastAPI()
app.add_middleware(InjectorMiddleware, injector=inj)
attach_injector(app, inj)

@app.get("/")
def get_root(b: B = Injected(B), b2: B = Injected(B)):
    return {"result": b.a, "result2": b2.a}
```

It causes following error:
```
Traceback (most recent call last):
  File "/.../python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 408, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/fastapi/applications.py", line 1106, in __call__
    await super().__call__(scope, receive, send)
  File "/.../python3.11/site-packages/starlette/applications.py", line 121, in __call__
    self.middleware_stack = self.build_middleware_stack()
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/fastapi/applications.py", line 1015, in build_middleware_stack
    app = cls(app=app, **options)
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/fastapi_injector/request_scope.py", line 175, in __init__
    self.request_scope_factory = injector.get(RequestScopeFactory)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/injector/__init__.py", line 91, in wrapper
    return function(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/injector/__init__.py", line 963, in get
    binding, binder = self.binder.get_binding(interface)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.../python3.11/site-packages/injector/__init__.py", line 659, in get_binding
    raise UnsatisfiedRequirement(None, interface)
injector.UnsatisfiedRequirement: unsatisfied requirement on RequestScopeFactory
```

How can I deal with that?

**Comment by ljnsn** [comment]
Indeed, it doesn't work, because `RequestScopeFactory` relies on auto binding. You can get it to work by adding a provider for it like so:

```python
from fastapi_injector import (
    RequestScope,
    RequestScopeFactory,
)

class MyModule(injector.Module):
    @injector.provider
    def provide_request_scope_factory(self, scope: RequestScope) -> RequestScopeFactory:
        """Provide the request scope factory."""
        return RequestScopeFactory(scope)
```

Then passing that module to the injector setup.

**Comment by matyasrichter** [comment]
Fixed in v0.5.4
